mobify/progressive-web-scaffold
# Use jQuery passed from jqueryResponse function

Sometimes jQuery would load slow. If our component tried to import jQuery before it loaded, an exception would be thrown becuase webpack builds a module that looks like this:

```
module.exports = jQuery;
```

That throws an exception if jQuery isn't defined. If we pull it from window, jQuery will just be undefined until it loads.

## Changes
- Grab jQuery from window

## TODO
- [ ] Release a new version of the SDK and bump the version in this PR

## How to test-drive this PR
- Download the scaffold
- Point the SDK to develop and `npm i && npm run dev`
- In Charles, enable SSH proxying for your local ip address
- Then open up `throttle settings` and throttle `https://<your-ip>:8443/static/js/jquery.min.js`
- Set the bandwidth to something ridiculous like 1kbps and the roundtrip time to 15000 ms
- Now preview merlins potions against your local dev server
- The page should load, and the title on the page shouldn't appear until jQuery loads, but it should eventually load.
